py/selenium/webdriver/phantomjs/service.py

```diff
@@ -75,13 +75,15 @@ def start(self):
                                             stdout=self._log, stderr=self._log)
 
         except Exception as e:
-            raise WebDriverException("Unable to start phantomjs with ghostdriver.", e)
+            raise WebDriverException(
```

Comment:
But why does it have to add the traceback to the msg?
It will be available with the exception itself..